src/lib.js

```diff
@@ -625,8 +666,16 @@ var program,
         return request;
       }
       catch (e) {
+        if (shouldRetry) {
+          try {
+            // Retry conversion again by considering cURL to be in windows cmd compatible format
+            return this.convertForCMDFormat(curlString);
+          }
+          catch (error) {
+            // Retry error is not reported
```

Comment:
Why are we ignoring this error instead of bubbling it up? What will the return value be in this case?

Reply:
@VShingala So if I understand this right, we don't actually have a mechanism to know if the user has copy-pasted a Windows curl command. We try the windows version if the bash one fails, and if the windows one also fails then we throw the original error. Right?

Reply:
@webholik Correct! We don't have detection mechanism and we just retry it. Same solution suggested by Abhijit is also mentioned in original JIRA issue.